src/test/java/com/course/work/democoursework/serviceTest/ExaminerServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package com.course.work.democoursework.serviceTest;
+
+import com.course.work.democoursework.Exception.QuantityRequestExceededException;
+import com.course.work.democoursework.Question;
+import com.course.work.democoursework.Service.JavaQuestionService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class ExaminerServiceImplTest {
+    @Mock
+    private JavaQuestionService mockJavaQuestionService;
+    @InjectMocks
+    private JavaQuestionService out;
```

Comment:
В этом тесте тестируем ExaminerServiceImpl.
Поэтому эту строку заменяем на
```suggestion
    private ExaminerServiceImpl out;
```
У ExaminerServiceImpl тут достаточно проверить метод getQuations